federation/develop/develop.sh

```diff
@@ -43,10 +43,15 @@ readonly TMP_DIR="$(mktemp -d)"
 readonly FEDERATION_OUTPUT_ROOT="${LOCAL_OUTPUT_ROOT}/federation"
 readonly VERSIONS_FILE="${FEDERATION_OUTPUT_ROOT}/versions"
 
-detect-project
-readonly KUBE_PROJECT="${KUBE_PROJECT:-${PROJECT:-}}"
+if [[ -n "${LOCAL_BUILD:-}" ]]; then
```

Comment:
I have no problems using KUBERNETES_PROVIDER.  That actually is better since it allows easier future expansion if special cases arise.

Change has been made